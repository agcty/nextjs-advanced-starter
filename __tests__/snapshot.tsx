import { render } from "@testing-library/react"

import Home from "../src/pages/index"

it("renders homepage unchanged", () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})
