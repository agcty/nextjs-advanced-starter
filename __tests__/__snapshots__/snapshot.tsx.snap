// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <main
      class="flex min-h-screen justify-center bg-gradient-to-b from-gray-50 via-gray-50 to-gray-100 py-20"
    >
      <div>
        <h1
          class="px-5 text-center text-4xl font-bold leading-tight tracking-tight sm:mt-4 sm:text-6xl"
        >
          Next.js
          <br />
          Advanced Starter
        </h1>
        <h2
          class="mx-auto mt-8 max-w-4xl px-10 text-center text-base tracking-tight text-gray-600 sm:text-2xl md:mt-5 md:text-2xl"
        >
          Tailwind CSS 3.0, ESLint & Prettier without a single line of config! Easily extendable zero-config template for pros and beginners.
        </h2>
        <div
          class="px-4 sm:px-0"
        >
          <section
            class="mt-6 grid min-h-[350px] w-full grid-cols-1 rounded-lg bg-white sm:mt-20 sm:min-w-[1000px] sm:grid-cols-2"
            style="box-shadow: rgba(0, 0, 0, 0.12) 0px 30px 60px 0px;"
          >
            <div
              class="flex flex-col justify-center rounded-l-lg bg-gray-50"
            >
              <ul
                class="space-y-5 px-12 py-12"
              >
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    Fast design workflow with 
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Tailwind CSS
                    </span>
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Tailwind CSS
                    </span>
                  </p>
                </li>
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      TypeScript
                    </span>
                     by default
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      TypeScript
                    </span>
                  </p>
                </li>
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    Customizable 
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      ESLint config
                    </span>
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      ESLint config
                    </span>
                  </p>
                </li>
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Code formatting
                    </span>
                     with Prettier
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Code formatting
                    </span>
                  </p>
                </li>
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    Standardized 
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      absolute imports
                    </span>
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Absolute imports
                    </span>
                  </p>
                </li>
                <li
                  class="flex items-center"
                >
                  <svg
                    class="hiddden hidden h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:inline"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M5 13l4 4L19 7"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                      stroke-width="2"
                    />
                  </svg>
                  <p
                    class="ml-3 hidden text-lg text-gray-600 sm:inline"
                  >
                    Ready-to-go 
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Jest
                    </span>
                     setup
                  </p>
                  <p
                    class="mx-auto sm:hidden"
                  >
                    <span
                      class="inline-flex items-center rounded-md bg-gray-100 px-3 py-2 font-medium text-gray-700"
                    >
                      <svg
                        class="mr-3 inline-flex h-5 w-5 flex-shrink-0 rounded-full bg-blue-600 p-1 text-gray-100 sm:hidden"
                        fill="none"
                        stroke="currentColor"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M5 13l4 4L19 7"
                          stroke-linecap="round"
                          stroke-linejoin="round"
                          stroke-width="2"
                        />
                      </svg>
                      Absolute imports
                    </span>
                  </p>
                </li>
              </ul>
            </div>
            <div
              class="space-y-5 place-self-center px-4 py-24 text-center"
            >
              <h3
                class="text-3xl font-bold"
              >
                Get it 👇
              </h3>
              <span
                class="inline-flex rounded-md shadow-sm"
              >
                <a
                  class="inline-flex items-center rounded-md border border-transparent bg-blue-600 px-4 py-4 font-medium leading-6 text-white transition duration-150 ease-in-out hover:bg-blue-500 focus:border-blue-700 focus:outline-none focus:ring-4 focus:ring-blue-400 active:bg-blue-700 sm:px-10"
                  href="https://github.com/agcty/nextjs-advanced-starter"
                  type="button"
                >
                  Copy Template from GitHub
                </a>
              </span>
            </div>
          </section>
          <p
            class="mt-6 text-center text-xs font-medium text-gray-600"
          >
            Built by
             
            <a
              class="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-500 focus:underline focus:outline-none"
              href="https://twitter.com/agctyz"
            >
              @agctyz
            </a>
          </p>
        </div>
      </div>
    </main>
  </div>
</div>
`;
